src/main/java/ru/hh/todolist/service/impl/TodoListServiceImpl.java

```diff
@@ -0,0 +1,95 @@
+package ru.hh.todolist.service.impl;
+
+import jakarta.inject.Inject;
+import jakarta.ws.rs.BadRequestException;
+import org.slf4j.Logger;
+import org.springframework.stereotype.Component;
+import ru.hh.todolist.dao.GenericDao;
+import ru.hh.todolist.dto.TaskDto;
+import ru.hh.todolist.entity.Task;
+import ru.hh.todolist.entity.TaskStatus;
+import ru.hh.todolist.mapper.TaskMapper;
+import ru.hh.todolist.service.TodoListService;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+import static org.slf4j.LoggerFactory.getLogger;
+
+@Component
+public class TodoListServiceImpl implements TodoListService {
+  private final GenericDao genericDao;
+
+  public static final Logger LOGGER = getLogger(TodoListServiceImpl.class);
+
+  @Inject
+  public TodoListServiceImpl(GenericDao genericDao) {
+    this.genericDao = genericDao;
+  }
+
+  @Override
+  public TaskDto add(String taskName) {
+    if (taskName == null || taskName.isBlank()) {
```

Comment:
В этих целях лучше использовать StringUtils:
```suggestion
    if (StringUtils.isBlank(taskName)) {
```

Reply:
посмотрел исходник, особого преимущества не увидел, ради эксперимента сделал. Простой isBlank() проверяет 
`if (ch != ' ' && ch != '\t' && !CharacterDataLatin1.instance.isWhitespace(ch)) {
`
не понимаю, зачем это двойная проверка.
 в StringUtils только 
  `if ((Character.isWhitespace(str.charAt(i)) == false)) {
  `